src/components/icon/icon.svelte

```diff
@@ -3,13 +3,23 @@
   import { writable } from 'svelte/store'
 
   let lastIconBasePath = '/icons'
-  let iconBasePath = writable(lastIconBasePath)
+  let iconBasePath = writable<{
```

Comment:
`iconBasePath` feels like the wrong name for this data structure. Maybe the following changes?
- `lastIconBasePath => defaultLibraryPath`
- `iconBasePath => iconLibraryPaths`